src/ert/gui/suggestor/_suggestor_message.py

```diff
@@ -133,9 +145,9 @@ def _expanded_text(self) -> str:
 
     def _text(self, location: str) -> str:
         return (
-            '<div style="font-size: 16px; line-height: 24px;">'
+            '<div style="font-size: 12px; line-height: 24px;">'
             + self._color_bold(self._header)
-            + self._message
+            + f"<pre>{self._message}</pre>"
```

Comment:
This changes the font to Courier for all messages, which is not really necessary for strings emitted by say the config parser.

Reply:
This relates to issue https://github.com/equinor/ert/issues/10892